modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -1819,7 +1819,7 @@ void LMDBBackend::getUpdatedPrimaries(vector<DomainInfo>& updatedDomains, std::u
 
     if (di.kind == DomainInfo::Producer) {
       catalogs.insert(di.zone);
-      catalogHashes[di.zone].process("\0");
+      catalogHashes[di.zone].process("");
```

Comment:
I know this change changes nothing, but what is the purpose of processing zero bytes?

Reply:
This will create catalogHashes[di.zone] for a catalog with no members. Without this there is no serial increase when you delete the last member from a catalog.